examples/pruning/lightgbm_integration.py

```diff
@@ -40,7 +39,7 @@ def objective(trial):
 
     # Add a callback for pruning.
     pruning_callback = optuna.integration.LightGBMPruningCallback(trial, 'binary_error')
-    gbm = lgb.train(param, dtrain, num_round, valid_sets=[dtest],
+    gbm = lgb.train(param, dtrain, valid_sets=[dtest],
```

Comment:
This line seems too short on the basis of Optuna's max line length, and I think it has room to move `verbose_eval=False,` from the next line to this line.

Reply:
I fixed this (commit: https://github.com/pfnet/optuna/pull/280/commits/7a2a6f44f5519e0e153c5a184056500386407058), thanks!